src/base/trafficGraph.js

```diff
@@ -670,7 +670,7 @@ class TrafficGraph extends EventEmitter {
     let changed = false;
     _.each(this.nodes, (node) => {
       node.defaultFiltered = !node.hasDefaultVisibleConnections() || !_.every(filters.node, filter => filter.passes(node, filter.defaultValue));
-      const filtered = !node.focused && (!node.hasVisibleConnections() || !_.every(filters.node, filter => filter.passes(node, filter.value)));
+      const filtered = !node.focused && !node.isEntryNode() && (!node.hasVisibleConnections() || !_.every(filters.node, filter => filter.passes(node, filter.value)));
```

Comment:
This can pose some problems with layout, as @aSqrd-eSqrd pointed out. I don't think we should _never_ filter out entry nodes.  The problem you are trying to solve, if I understand it correctly, is that you have a graph with a single node and no connections.

To solve your use case, I think adding a check if the graph only has one node, don't filter it out.